ArduCopter/toy_mode.h

```diff
@@ -97,6 +99,13 @@ class ToyMode
         BLINK_VSLOW  = 0xF000,
         BLINK_MED_1  = 0xF0F0,
     };
+    
+    enum takeoff_states {
+        TAKEOFF_ON_GROUND   = 0,
```

Comment:
Maybe `TAKEOFF_INACTIVE`.  You may be on the ground and spinning props menacingly.